src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -131,13 +131,15 @@
      * @param connector
      *      HttpConnector to use. Pass null to use default connector.
      */
-    /* package */ GitHub(String apiUrl, String login, String oauthAccessToken, String password, HttpConnector connector, RateLimitHandler rateLimitHandler, AbuseLimitHandler abuseLimitHandler) throws IOException {
+    /* package */ GitHub(String apiUrl, String login, String oauthAccessToken, String applicationToken, String password, HttpConnector connector, RateLimitHandler rateLimitHandler, AbuseLimitHandler abuseLimitHandler) throws IOException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 8 parameters, which is greater than 7 authorized. ([squid:S00107](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.github%3Agithub-api&pullRequest=4&issues=AWMk6_Pwcd1riAyNJJdu&open=AWMk6_Pwcd1riAyNJJdu)

[ISSUE_KEY]: AWMk6_Pwcd1riAyNJJdu